examples/configs/sunk-tcp.yaml

```diff
@@ -1,8 +1,10 @@
-- examples.devices.remote_controlled.RemoteControlled:
-    name: tcp_contr
-    inputs: {}
-    format: "%b\r\n"
-- tickit.devices.sink.Sink:
-    name: contr_sink
-    inputs:
-      input: tcp_contr:observed
+- type: examples.devices.remote_controlled.RemoteControlled
+  name: tcp_contr
+  inputs: {}
+  format: "%b\r\n"
+- type: tickit.devices.sink.Sink
+  name: contr_sink
+  inputs:
+    input:
+      component: tcp_contr
+      port: observed
```

Comment:
This one throws a `ValidationError` when starting

Reply:
Resolved, another minor config change required, from format: string to format: dict for Pydantic to be happy deserialising it.